### DISCUSSION TITLE
Ctrl + Click and Drag to copy instance

### DISCUSSION THREAD

**Discussion by talmo** [talmolab/sleap#1846] [Ideas]
This functionality would mimic the behavior in PowerPoint where we can hold control + click and drag an object to instantly copy it and move the whole object.

It would be equivalent to copy-pasting an instance in SLEAP and while the mouse is still pressed down, keep it in whole-instance-moving mode (like Alt+click and drag).

**Comment by gitttt-1234** [comment]
Closing this discussion, added this feature in https://github.com/talmolab/sleap/pull/2072
